composer.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "alleyinteractive/wp-match-blocks",
+    "description": "Match WordPress blocks in the given content.",
+    "type": "library",
+    "license": "GPL-2.0-or-later",
+    "authors": [
+        {
+            "name": "Alley",
+            "email": "info@alley.co"
+        }
+    ],
+    "autoload": {
+	  "classmap": [
+		"src/"
+	  ],
+	  "files": [
+		"src/Alley/WP/match-blocks.php",
+		"src/Alley/WP/Internals/internals.php"
+	  ]
+    },
+    "config": {
+        "optimize-autoloader": true,
+        "classmap-authoritative": true,
+        "allow-plugins": {
+            "alleyinteractive/composer-wordpress-autoloader": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
+    },
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "alleyinteractive/laminas-validator-extensions": "dev-feature/supporting-validators"
```

Comment:
This will be switched to `^1.1` after https://github.com/alleyinteractive/laminas-validator-extensions/pull/2 is merged and tagged.